donsus_test/parser/test_if_statements.cc

```diff
@@ -5,7 +5,7 @@
 TEST(IfStatements, IfStatementsNodeType) {
   std::string a = R"(
         def a() -> int {
-            if() {}
+            if(1) {}
```

Comment:
Does this refer to the fact that some of the expressions can be expressed as booleans so that when we specify 1 we mean true, can you explain the idea behind this addition.